src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -23,10 +23,7 @@
 import com.github.bartimaeusnek.bartworks.common.loaders.ItemRegistry;
 
 import gregtech.api.enums.*;
-import gregtech.api.util.GT_ModHandler;
-import gregtech.api.util.GT_OreDictUnificator;
-import gregtech.api.util.GT_Recipe;
-import gregtech.api.util.GT_Utility;
+import gregtech.api.util.*;
```

Comment:
Please replace wildcart import with multiple regular ones.

Reply:
This has been fixed now. I think IntelliJ did this automatically or something at some point.